airbyte-webapp/build.gradle

```diff
@@ -27,6 +28,3 @@ task test(type: NpmTask) {
     inputs.file 'package-lock.json'
 }
 
-clean {
```

Comment:
I don't think this is correct to do that. The clean should give us the guarantee that we're starting from a clean slate.

Maybe the solution is to not call clean when we build in CI

Reply:
OK then, I am sold!